sites/public/src/components/shared/FormSummaryDetails.tsx

```diff
@@ -193,16 +195,17 @@ const FormSummaryDetails = ({
 
       <Card.Section className={styles["summary-section"]}>
         <FieldValue
-          testId={"app-summary-name"}
+          testId={"app-summary-applicant-name"}
           id="applicantName"
           label={t("t.name")}
           className={styles["summary-value"]}
         >
-          {application.applicant.firstName} {application.applicant.middleName}{" "}
-          {application.applicant.lastName}
+          {`${application.applicant.firstName}${
```

Comment:
We were showing some empty spaces here even if the user did not enter a middle name